src/components/charts/OutboundAlert.js

```diff
@@ -0,0 +1,181 @@
+import React from "react"
+import Grid from "@material-ui/core/Grid"
+import Paper from "@material-ui/core/Paper"
+import Link from "@material-ui/core/Link"
+import Typography from "@material-ui/core/Typography"
+import outboundIcon from "@components/icons/outbound.png"
+import { getCountryFromISO } from "@/utils/mapBaiduCountry"
+import { withStyles } from "@material-ui/core/styles"
+import Tooltip from "@material-ui/core/Tooltip"
+import styled from "styled-components"
+import { useTranslation } from "react-i18next"
+import { withLanguage, getLocalizedPath } from "@/utils/i18n"
+import _groupBy from "lodash.groupby"
+
+const StyledTooltip = withStyles({
```

Comment:
is this possible to use styled-component as well?

Reply:
applied